libs/indibase/inditelescope.h

```diff
@@ -781,6 +789,20 @@ class Telescope : public DefaultDevice
 
         const char * getPierSideStr(TelescopePierSide ps);
 
+        // Satellite tracking
+        IText TLEtoTrackT[1] {};
+        ITextVectorProperty TLEtoTrackTP;
+        IText SatPassWindowT[2] {};
+        ITextVectorProperty SatPassWindowTP;
+        enum SATELLITE_TRACKING
+        {
+            SAT_TRACK,
```

Comment:
Please document the enum using Doxygen style coding.

Also, I think the class itself should probably document how this is used with a simple example. This is to allow prospective developers from understand what these properties are exactly for and how to utilize them in their own drivers.

Reply:
Sure! Is a docstring fine regarding the example? Or is it maybe better to just point at the implementation in the LX200 so they see how it was done in that case?

Reply:
I went ahead and added docstrings with a `\ref` to the LX200_10micron implementation. Let me know if you think that is sufficient